public/lib/helpers/timeline_helper.js

```diff
@@ -122,6 +122,22 @@ define(function (require) {
       return sortObj;
     };
 
+    /**
+     * Creates an Elasticsearch sort object to sort in chronological order on end field
+     *
+     * @param params group configuraton parameters
+     * @returns Elasticsearch sort object
+     */
+    TimelineHelper.prototype.getSortOnEndFieldObject = function (params) {
```

Comment:
This method is completely equal to the `getSortOnStartFieldObject` method above. Don't duplicate the code, use one method instead.